F-FrontEnd/src/F-intrinsics-table.c

```diff
@@ -1617,7 +1617,7 @@ intrinsic_entry intrinsic_table[] = {
     {INTR_SELECTED_REAL_KIND,
      INTR_NAME_GENERIC,
      "selected_real_kind",
-     {INTR_TYPE_INT, INTR_TYPE_INT, INTR_INT},
+     {INTR_TYPE_INT, INTR_TYPE_INT, INTR_TYPE_INT},
```

Comment:
This is part of "Made c++ compatibility fixes to the F_front code" (from description of this PR). There was an error here, as "INTR_INT" is from entirely different enum. I guess it went unreported, because  C compilers are more lax about type safety.